blocks/api/test/paste.js

```diff
@@ -45,3 +48,44 @@ describe( 'normaliseToBlockLevelNodes', () => {
 	} );
 } );
 
+describe( 'paste', () => {
+	registerBlockType( 'test/small', {
```

Comment:
Since the block registry is global, we should make sure to reset to its original state after our tests are done.

Example from serializer tests:

https://github.com/WordPress/gutenberg/blob/77620b2b823e6ecf856e782eac580987983c9ae0/blocks/api/test/serializer.js#L20-L24